doc/userguide/src/ExtendingRobotFramework/CreatingTestLibraries.rst

```diff
@@ -1424,11 +1433,10 @@ Framework.
        logging.info('This is a boring example')
 
 The `logging` module has slightly different log levels than
-Robot Framework. Its levels `DEBUG` and `INFO` are mapped
-directly to the matching Robot Framework log levels and `WARNING`
-and everything above is mapped to `WARN`. Custom levels below
-`DEBUG` are mapped to `DEBUG` and everything between
-`DEBUG` and `WARNING` is mapped to `INFO`.
+Robot Framework. Its levels `DEBUG`, `ERROR`, and `INFO` are mapped
+directly to the matching Robot Framework log levels. The `WARNING`
+log level is mapped to `WARN` and `CRITICAL` is mapped to `ERROR`. Custom levels
+above `ERROR` are mapped to `ERROR` Custom levels below `DEBUG` are mapped to `DEBUG` and everything between `DEBUG` and `WARNING` is mapped to `INFO`.
```

Comment:
The original documentation was already a bit complicated and adding ERROR into the mix makes it worse. Perhaps it would be better to say something like this:

> Its levels `DEBUG` , `INFO`, `WARNING` and `ERROR` are mapped directly to the matching Robot Framework log levels and `CRITICAL` is mapped to `ERROR. Possible custom levels are mapped to the closest standard level. For example, a custom level between `INFO` and `WARNING` is consider `INFO`.